README.md

```diff
@@ -1,51 +1,83 @@
+| `techLead` | `boolean` | unused | is team member able to serve as a tech lead |
+| `skillLevel` | `number` | unused | |
+| `active` | `boolean` | unused | |
+| `cohortLeader` | `string` | optional | cohort team member is leader of, if they lead one | 
+| `cohortParticipant` | `string` | optional | cohort team member is participant in |
+| `internalResponsibility` | `string[] ` | unused | |
+| `discoveryLead` | `boolean` | unused | |
+| `certifications` | `string[]` | optional | team members's certifications, if they have any. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"PMP, PSM"` |
+| `bio` | `string` | optional | team member's bio |
+| `funFacts` | `string[]` | **required** | fun facts about the team member. can be [] if none |
+| `technologies` | `string[]` | optional | technologies a team member is proficient in.`string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `expertise` | `string[]` | optional | technologies or skill sets a team member has expertise in. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `strengths` | `string[]` | optional | team member's strengths. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `interests` | `string[]` | optional | team member's interests. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `currentProjects` | `project[]` | optional | current projects the team member is on. [See Below for project data specification](#project-data) |
+| `pastProjects`| `string[]` | **required** | past projects a team member has worked on |
+| `photo` | `string` | **required** | url of team member's photo |
+| `weeklyCapacity` | `number` | **required** | weekly capacity of maximum billable hours for a team member |
+| `forecastedHours` | `number` | **required** | forecasted weekly billable hours for a team member |
+| `pto` | `number` | unused | |
+
+
+#### Project Data
+
+Data for each project is represented in a JSON object. Each data object should contain the following properties. 
+
+| Property | Type | Status | Description |
+| ------- | ------- | ------- | ------- |
+| `client`| `string`| **required** | name of client |
+| `project` | `string`| **required** | name of project |
+| `startDate` | `"YYYY-MM-DD"`|  **required** | date of project start *ex* `"2021-04-12"` | 
+| `endDate` | `"YYYY-MM-DD"`|  **required** | date of project start *ex* `"2021-04-12"` | 
+
+
+### Browser Testing
 <!--
 The browser list should be tailored to specific engagement and client needs.
 Delete if irrelevant to this issue
 -->
 
-## Gold Level Support
+#### Gold Level Support
```

Comment:
It's not related to your work here, but we can delete the comment starting in line 73 – I think that was copied from the PR template at some point.

Reply:
And thanks for fixing up the heading levels!